src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -140,15 +141,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String messageContent = request.getParameter("message");
 
-    // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
+    /* Can be used if we would rather have Markdown for stylizing text. Doesn't currently
+    pass test because it cleans up html slightly differently (just gets rid of the not allowed
+    html tags, not the content inside of the tags). I think the test can just
+    be edited if this is what we want */
+
+    //TextProcessor processor = BBProcessorFactory.getInstance().createFromResource(ConfigurationFactory.MARKDOWN_CONFIGURATION_FILE);
```

Comment:
You shouldn't leave code commented out when committing. If you don't need this code, delete it! :)